File: trader/world.rb
require 'player'
require 'city'

class World

  attr_reader :player, :cities


  def initialize
    @cities = [City.new, City.new]
    @cities[0].add_connection(@cities[1], 25)
    @player = Player.new(@cities[0])
  end


end

File: _old/trader/old/world.cc
#include "world.h"

#include "player.h"
#include "city.h"

World::World()
	: player(player)
{
	City *c1 = new City(), *c2 = new City();
	c1->AddConnection(c2, 100);
	cities.push_back(c1);
	cities.push_back(c2);
}


World::~World()
{
}
